patches/net/minecraft/client/color/block/BlockColors.java.patch

```diff
@@ -5,8 +5,8 @@
  public class BlockColors {
     private static final int DEFAULT = -1;
 -   private final IdMapper<BlockColor> blockColors = new IdMapper<>(32);
-+   // FORGE: Use registry delegate as non-Vanilla block ids are not constant
-+   private final java.util.Map<net.minecraft.core.Holder.Reference<Block>, BlockColor> blockColors = new java.util.HashMap<>();
++   // NEOFORGE: Use the block instance directly as non-Vanilla block ids are not constant
```

Comment:
The syntax for this comment should be `Neo:` not `NEOFORGE:`

Reply:
Replaced by `// Neo:`